test/docs/OpenJ9TestUserGuide.md

```diff
@@ -0,0 +1,302 @@
+    exclude MyTest from running on the aix platform, for example:
+    ```
+    @Test(groups={ "level.sanity", "component.jit", "disabled.os.aix" })
+        public class MyTest {
+        ...
+    ```
+
+We currently support the following exclusion groups:
+```
+disabled.os.<os> (e.g. disabled.os.aix)
+disabled.arch.<arch> (e.g. disabled.arch.ppc)
+disabled.bits.<bits> (e.g. disabled.bits.64)
+disabled.spec.<spec> (e.g. disabled.spec.linux_x86-64)
+```
+
+6. View results:
+
+  * in the console
+
+    OpenJ9 tests take advantage of the testNG logger.
+    If you want your test to print output, you are required to use the
+    testng logger (and not System.out.print statements). In this way,
+    we can not only direct that output to console, but also to various
+    other clients (WIP).  At the end of a test run, the results are
+    summarized to show which tests passed / failed / skipped.  This gives
+    you a quick view of the test names and numbers in each category
+    (passed/failed/skipped).  If you've piped the output to a file, or
+    if you like scrolling up, you can search for and find the specific
+    output of the tests that failed (exceptions or any other logging
+    that the test produces).
+
+  * in html files
+
+    Html (and xml) output from the tests are created and stored in a
+    test_output_xxxtimestamp folder in the TestConfig directory (or from
+    where you ran "make test").  The output is organized by tests, each
+    test having its own set of output.  If you open the index.html file
+    in a web browser, you will be able to see which tests passed, failed
+    or were skipped, along with other information like execution time and
+    error messages, exceptions and logs from the individual test methods.
+
+  * TAP result files
+
+    TAP files are created. Depending on the requirement there are three
+    different diagnostic levels.
+    
+		* export DIAGNOSTICLEVEL=failure
+			Default level. Log all detailed failure information if test fails
+		* export DIAGNOSTICLEVEL=all
+			Log all detailed information no matter test failures or succeed
+		* export DIAGNOSTICLEVEL=nodetails
+			No need to log any detailed information. Top level TAP test result
+			summary is enough 
+			
+  * in the cloud (WIP)
+  
+
+7. Attach a debugger:
+
+  * to a particular test
+
+    The command line that is run for each particular test is echo-ed to
+    the console, so you can easily copy the command that is run.
+    You can then run the command directly (which is a direct call to the
+    java executable, adding any additional options, including those to
+    attach a debugger.
+
+8. Move test into different make targets (layers):
+
+  * from extended to sanity (or vice versa)
+
+    Change the group annotated at the top of the test class from
+    "level.extended" to "level.sanity" and the test will be automatically
+    switched from the extended target to the sanity target.
```

Comment:
There are 2 parts, there is tagging the testng class, and the tag in the playlist.  Are both places needed to change...